crates/runtime/src/embeddings/connector.rs

```diff
@@ -357,17 +355,37 @@ impl DataConnector for EmbeddingConnector {
         let underlying_table = Arc::clone(&embedding_table.base_table);
         let underlying_federated_table = Arc::new(FederatedTable::Immediate(underlying_table));
 
-        #[cfg(feature = "debezium")]
         let stream = self
             .inner_connector
             .changes_stream(underlying_federated_table)?
             .then(move |item| Self::embed_change_envelope(item, Arc::clone(&embedding_table)))
             .boxed();
 
-        #[cfg(not(feature = "debezium"))]
```

Comment:
Flag is not required anymore as we moved `replace_change_batch_data` into cdc module.